### ISSUE TITLE
Entity::toArray error at PostgreSQL

### ISSUE THREAD

**Issue by f3l1x** [nextras/orm#156]
Version: v1.1.6, maybe also in 2.0

```php
$entity->toArray($entity::TO_ARRAY_RELATIONSHIP_AS_ID);
```

It generates:

```
SELECT order.* FROM "order" AS "order" WHERE order.customer_id IN (1)
```

Error: `ERROR: syntax error at or near "order"
LINE 1: SELECT order.* FROM "order" AS "order" WHERE "order"."custom...`

But it should be:

```
SELECT "order".* FROM "order" AS "order" WHERE "order"."customer_id" IN (1)
```

----

Any idea?

**Comment by hrach** [comment]
Should be already fixed in 2.0. https://github.com/nextras/orm/commit/9a47965141ca05d305b76724469747d58aeed5c7

**Comment by f3l1x** [comment]
Greate. I have to stay at 1.6 at least a few weeks. Could you please fix it to ~1.1?

**Comment by hrach** [comment]
Yeah. That's why it's marked as a bug.

**Comment by hrach** [comment]
I see that the commit is already included in v1.1. So, are you sure you have the correct version?

**Comment by f3l1x** [comment]
You're right, I had `nette/cache:~2.3.0`. It's fixed at 2.4?

**Comment by hrach** [comment]
Wha't have nette caching common with this issue?

**Comment by f3l1x** [comment]
I mean this - https://github.com/nextras/orm/issues/143#issuecomment-160193621

**Comment by f3l1x** [comment]
I have had 1.1.6 version, I'm sure. At 2.0 it's oukey.

**Comment by hrach** [comment]
fixed in 1.1.7
